client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
+import Logo from '../assets/logo.png'
+import Search from '../assets/search.png';
+import Mic from '../assets/mic.png'
+import Suggestion from "./data";
+import data from './data'
+class SearchField extends Component{
+
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            words: [],
+            userInput: '',
+            matches: [],
+            rowHighlighted: -1
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+            words: data.words
+        })
+    }
+
+    handleUserInput(e) {
```

Comment:
Any class method that is not a lifecycle hook should be defined as an arrow function.
This makes code more readable (since I instantly know that this particular function does not come with React and is defined by the developer). It also prevents unusual stuff from happening because of JavaScript's `this` binding.

So, change this to an arrow function.